docs/designers-developers/designers/menu-item.md

```diff
@@ -0,0 +1,151 @@
+# MenuItem
+
+MenuItem is a component which renders a button intended to be used in combination with the DropdownMenu Component.
+
+![An image of a MenuItem being highlighted inside of a DropdownMenu component](https://wordpress.org/gutenberg/files/2018/11/MenuItem.png)
+
+1. MenuItem
+
+
+
```

Comment:
I see a lot of empty lines, Shouldn't we just separate sections with a single empty line?

Reply:
Yep! Sounds great. I'll update that.